pkg/package/manifest.go

```diff
@@ -210,6 +236,11 @@ func parseVariableSchema(manifest []byte) map[string]Schema {
 			if description, okk := vv["description"].(string); okk {
 				schema.Description = description
 			}
+
+			if dflt, okk := vv["default"]; okk {
+				b, _ := json.Marshal(dflt)
```

Comment:
I probably haven't done a great job explaining this one but I think we should allow the user to define yaml values with type int, but something like `controlplane_count: 3` unmarshals as int, so it will fail a string cast (so we can't store it in Default), which is why we need to marshal it back to a string.

Reply:
ahh that makes sense, in that case what you have here makes sense.